### PR TITLE
Make minimum graph height configurable

### PR DESCRIPTION
**Pull request by pblasquez**

### CODE REVIEW

File: includes/defaults.inc.php [hunk 1]
```diff
@@ -124,6 +124,7 @@ function set_debug($debug) {
 // This enables the top X ports box
 $config['top_devices'] = 1;
 // This enables the top X devices box
```
Comment (paulgear) [inline includes/defaults.inc.php]: Can we get this comment line moved to where it should be, or removed?

Comment (paulgear) [inline includes/defaults.inc.php]: Scratch that - looks like it's below rather than above.

**Comment by laf** [comment]
This needs to be a webui config option.

**Comment by pblasquez** [comment]
Ah, missed that option, will research and modify.

**Pull request merged**